src/utils/daos.c

```diff
@@ -1389,11 +1362,7 @@ help_hdlr(int argc, char *argv[], struct cmd_args_s *ap)
 		if (argc == 3) {
```

Comment:
minor: since you did that you can just:
if (argc == 3 || strcmp(argv[3], "copy") == 0)
  FS_COPY_CMDS_HELP();